tests/unit_tests/weightwindows/test_ww_gen.py

```diff
@@ -229,9 +229,9 @@ def test_ww_gen_roundtrip(run_in_tmpdir, model):
     particle_type = 'neutron'
 
     wwg = openmc.WeightWindowGenerator(mesh, energy_bounds, particle_type)
-    wwg.update_params = {'ratio' : 5.0,
-                         'threshold': 0.8,
-                         'value' : 'mean'}
+    wwg.update_parameters = {'ratio' : 5.0,
+                            'threshold': 0.8,
+                            'value' : 'mean'}
```

Comment:
I think these two lines should be indented one space more and the tab or white space in front removed, thanks for the fix!

Reply:
You should be able to set your text editor to convert tabs to 4 spaces for python files and [PEP 8](https://peps.python.org/pep-0008/#tabs-or-spaces) suggests to use spaces instead.